code/controllers/subsystem/ticker.dm

```diff
@@ -648,7 +648,8 @@ SUBSYSTEM_DEF(ticker)
 		'sound/roundend/ssethenjoyedyourchaos.ogg',
 		'sound/roundend/ssethyoumakemesick.ogg',
 		'sound/roundend/reasonsunknown.ogg',
-		'sound/roundend/moon.ogg'\
+		'sound/roundend/moon.ogg',
+		'sound/roundend/wings_of_redemption.ogg'\
```

Comment:
rename `wings_of_redemption.ogg` to `wings of re(dem/dey)ption.ogg` okay honey 💅

Reply:
![soyjacking off](https://user-images.githubusercontent.com/54416597/168568963-d42202db-7fb7-465f-af0a-3e72818e051c.png)